/**:
  ros__parameters:
    common:
      sensor_height: 1.723
      min_range: 0.27
      max_range: 80.0
      seed_selection_weight: -1.1
    rnr: # Reflected Noise Removal
      num_sample: 20
      min_height_threshold: -2.123 # init = -sensor_height + height_noise_margin
      min_vertical_angle_threshold: -15.0
      min_intensity_threshold: 0.2
    czm: # Concentric Zone Model
      num_zone: 4
      min_num_points: 10
      min_zone_ranges: [2.7, 12.3625, 22.025, 41.35]
      num_sectors: [16, 32, 54, 32]
      num_rings: [2, 4, 4, 4]
      elevation_thresholds: [0.0, 0.0, 0.0, 0.0]
      flatness_thresholds: [0.0, 0.0, 0.0, 0.0]
    rpf: # Region-wise Plane Fitting (R-VPF & R-GPF)
      max_vertical_distance_threshold: 0.1
      max_angle_threshold: 0.707
      max_distance_threshold: 0.1
      num_iterator: 3
      num_sample: 20
    gle: # Adaptive Ground Likelihood Estimation
      elevation_std_weights: [1.0, 1.0, 1.0, 1.0]
      flatness_std_weights: [1.0, 2.0, 2.0, 2.0]
      height_noise_margin: -0.5
    tgr: # Temporal Ground Revert
      std_weight: 1.5
